src/progressbar/docs/readme.md

```diff
@@ -1,32 +1,65 @@
 A progress bar directive that is focused on providing feedback on the progress of a workflow or action.
 
-It supports multiple (stacked) bars into the same `<uib-progress>` element or a single `<progressbar>` element with optional `max` attribute and transition animations.
+It supports multiple (stacked) `<uib-bar>` into the same `<uib-progress>` element or a single `<uib-progressbar>` element with optional `max` attribute and transition animations.
 
-### Settings ###
+### uib-progressbar settings
 
-#### `<uib-progressbar>` ####
+* `value`
+  <small class="badge">$</small>
+  <i class="glyphicon glyphicon-eye-open"></i> -
+  The current value of progress completed.
 
- * `value` <i class="glyphicon glyphicon-eye-open"></i>
- 	:
- 	The current value of progress completed.
+* `type`
+  _(Default: `null`)_ -
+  Style type. Possible values are 'success', 'warning' etc.
 
- * `type`
- 	_(Default: null)_ :
- 	Style type. Possible values are 'success', 'warning' etc.
+* `max`
+  <small class="badge">$</small>
+  <small class="badge">C</small>
+  <i class="glyphicon glyphicon-eye-open"></i>
+  _(Default: `100`)_ -
+  A number that specifies the total value of bars that is required.
 
- * `max`
- 	_(Default: 100)_ :
- 	A number that specifies the total value of bars that is required.
+* `animate`
+  <small class="badge">$</small>
+  <small class="badge">C</small>
+  _(Default: `true`)_ -
+  Whether bars use transitions to achieve the width change.
 
- * `animate`
- 	_(Default: true)_ :
- 	Whether bars use transitions to achieve the width change.
+* `title`
+  _(Default: `progressbar`)_ -
+  Title to use as label (for accessibility).
+  
+### uib-progress settings
 
- * `title`
-  _(Default: progressbar)_ :
-  Title to use as label (for accessibility)
+* `max`
+  <small class="badge">$</small>
+  <small class="badge">C</small>
+  <i class="glyphicon glyphicon-eye-open"></i>
+  _(Default: `100`)_ -
+  A number that specifies the total value of bars that is required.
 
-### Stacked ###
+* `animate`
+  <small class="badge">$</small>
+  <small class="badge">C</small>
+  _(Default: `true`)_ -
+  Whether bars use transitions to achieve the width change.
 
-Place multiple `<uib-bar>`s into the same `<uib-progress>` element to stack them.
-`<uib-progress>` supports `max`, `animate`, and `title` &  `<uib-bar>` supports  `value`, `title`, and `type` attributes.
+* `title`
+  _(Default: `progressbar`)_ -
+  Title to use as label (for accessibility).
+  
+### uib-bar settings
+
+* `value`
+  <small class="badge">$</small>
+  <i class="glyphicon glyphicon-eye-open"></i> -
+  The current value of progress completed.
+
+* `type`
+  _(Default: `null`)_ -
+  Style type. Possible values are 'success', 'warning' etc.
```

Comment:
Maybe change to `Bootstrap style type. Possible values are 'success', 'info', 'warning', and, 'error' to use Bootstrap's pre-existing styling, or any desired custom suffix`, or something that details this a little more.